frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/ArchiveExtractor.java

```diff
@@ -139,4 +141,46 @@ public void extract(String archive, String destinationDirectory) throws ArchiveE
                     + "'", e);
         }
     }
+
+    /**
+     * Do multiple file system checks that should enable the plugin to work on any file system
+     * whether or not it's case sensitive or not.
+     *
+     * @param destPath
+     * @param destDir
+     * @return
+     */
+    private boolean startsWithPath(String destPath, String destDir) {
+        if (destDir.startsWith(destDir)) {
```

Comment:
Won't this check always be true? Is it supposed to be `destPath`?

Reply:
That was a typo on my part.